app/src/main/java/com/cornellappdev/uplift/ui/nav/BottomNavScreen.kt

```diff
@@ -0,0 +1,37 @@
+package com.cornellappdev.uplift.ui.nav
+
+import com.cornellappdev.uplift.R
+import com.cornellappdev.uplift.ui.UpliftRootRoute
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class BottomNavScreens<T>(
+    val route: T,
+    val painterIds: Pair<Int, Int>,
+    val titleText: String
+) {
```

Comment:
Oh weird wait sorry im a bit confused.

I saw before that you had the `data class UpliftRootRoute` for nav routes; what's the difference between that and this? I would've thought that that class would replace this one.

Oh, my guess is that this is just used for the nav bar at the bottom to get stuff to render. Is that right?

Reply:
Yep it was mainly for the nav bar at the bottom. When I was updating, I think there was originally a BottomNavScreen class, and I wasn't sure if it would be better to separate or combine into the UpliftRootRoute class.